addons/purchase/invoice.py

```diff
@@ -35,8 +35,8 @@ def _prepare_invoice_line_from_po_line(self, line):
             qty = line.product_qty - line.qty_invoiced
         else:
             qty = line.qty_received - line.qty_invoiced
-        if float_compare(qty, 0.0, precision_rounding=line.product_uom.rounding) <= 0:
-            # If received quantities or quantities is zero is better not create invoice lines
+        if float_compare(qty, 0.0,
+                         precision_rounding=line.product_uom.rounding) == 0:
             return {}
```

Comment:
@sergio-teruel I agree woth you that this would be much better! Yes, please create a PR